util/cron/common-homebrew.bash

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# This script will clone the home-brew repository under test and copies the chapel formula in chapel-lang repo under
```

Comment:
Could there be a sentence about the overall goal here?  Is it to test our current homebrew formula on the local repository?  Is it to test a new formula on the local clone?  Is it running the same things run by the homebrew CI/CD?

Reply:
The answer is 
"Is it to test our current homebrew formula on the local repository?" Added a comment to reflect this.